_config/site.yml

```diff
@@ -141,8 +141,10 @@ profiles:
   # Docker staging configuration (PR and Nightly)
   staging_docker:
     <<: *profile
-    dcp_base_url: http://searchisko:8080/
-    dcp_base_protocol_relative_url: //<%= ENV['ACCESSIBLE_SLAVE_IP'] %>:<%= ENV['SEARCHISKO_HOST_PORT'] %>/
+    dcp_base_url: http://dcp.stage.jboss.org/
+    dcp_base_protocol_relative_url: //dcp.stage.jboss.org/
+#    dcp_base_url: http://searchisko:8080/
+#    dcp_base_protocol_relative_url: //<%= ENV['ACCESSIBLE_SLAVE_IP'] %>:<%= ENV['SEARCHISKO_HOST_PORT'] %>/
```

Comment:
I'll back it out after merging this into master. Otherwise I'll net have a passing build.